src/resources/objects/interfaces.ts

```diff
@@ -31,10 +31,13 @@ export interface ListObjectOptions extends PaginationOptions {
   name?: string;
 }
 
-export interface ListObjectSubscriptionsOptions extends PaginationOptions {}
+export interface ListObjectSubscriptionsOptions extends PaginationOptions {
+  mode?: string;
```

Comment:
I don't think we need this here given we set it on the `getSubscriptions` method?